### PR TITLE
Feat/2346 entities concepts explore records

### PR DESCRIPTION
**Pull request by kgish**

### CODE REVIEW

File: app/views/entities/show.rb [hunk 1]
```diff
@@ -83,6 +73,31 @@ def content
       end
     end
 
+    def entity_tab_items
```
Comment (rwd) [inline app/views/entities/show.rb]: Similar code is repeated in these hashes, "data.europeana.eu" is repeated in `#entity_query_items_by` and `#entity_query_items_by` only supports one tab.

I recommend replacing `#entity_tab_items`, `#entity_query_items_by` and `#build_proxy_dc` with:
```ruby
    ENTITY_SEARCH_QUERY_FIELDS = {
      agent: {
        by: %w(proxy_dc_creator proxy_dc_contributor)
      },
      concept: {
        about: 'what'
      }
    }.freeze

    def entity_tab_items
      ENTITY_SEARCH_QUERY_FIELDS[api_type].keys.map do |relation|
        entity_tab_items_one_tab(api_type, relation)
      end
    end

    def entity_tab_items_one_tab(api_type, relation)
      search_query = entity_search_query(api_type, relation)
      {
        tab_title: t("site.entities.tab_items.items_#{relation}", name: entity_name),
        url: search_path(q: search_query, format: 'json'),
        search_url: search_path(q: search_query)
      }
    end

    def entity_search_query(api_type, relation)
      fields = ENTITY_SEARCH_QUERY_FIELDS[api_type.to_sym][relation.to_sym]
      [fields].flatten.map do |field|
        %(field: "#{http://data.europeana.eu/#{api_path}}")
      end.join(' OR ')
    end
```

Then we will only need to modify `ENTITY_SEARCH_QUERY_FIELDS` when adding tabs.

File: app/views/concerns/entity_displaying_view.rb [hunk 2]
```diff
@@ -58,21 +59,35 @@ def entity_name
...
-    entity_value_by_locale(@entity[:biographicalInformation])
+    api_type == 'agent' ? entity_value_by_locale(@entity[:biographicalInformation]) : entity_note(@entity[:note])
   end
 
   private
 
+  # Returns a string: locale, english or nil
+  def entity_note(note)
+    return nil unless note && note.is_a?(Hash) && note.length.positive?
+    note.each do |key, value|
```
Comment (kgish) [inline app/views/concerns/entity_displaying_view.rb]: Nice and compact, and more readable (for high IQ's) ...

File: app/views/concerns/entity_displaying_view.rb [hunk 3]
```diff
@@ -58,21 +59,35 @@ def entity_name
...
   def entity_description
-    entity_value_by_locale(@entity[:biographicalInformation])
+    api_type == 'agent' ? entity_value_by_locale(@entity[:biographicalInformation]) : entity_note(@entity[:note])
   end
 
   private
 
+  # Returns a string: locale, english or nil
+  def entity_note(note)
+    return nil unless note && note.is_a?(Hash) && note.length.positive?
```
Comment (kgish) [inline app/views/concerns/entity_displaying_view.rb]: Yeps.

File: app/views/entities/show.rb (hunk 1 above)
Comment (kgish) [inline app/views/entities/show.rb]: Nice and elegant, `%(field: "#{http://data.europeana.eu/#{api_path}}")` should be `%(field: "http://data.europeana.eu/#{api_path}}")`.

Comment (kgish) [inline app/views/entities/show.rb]: Also, first `api_type` => `api_type.to_sym`

Comment (kgish) [inline app/views/entities/show.rb]: It works!

File: app/views/entities/show.rb [hunk 4]
```diff
@@ -83,15 +73,48 @@ def content
       end
     end
 
+    ENTITY_SEARCH_QUERY_FIELDS = {
```
Comment (rwd) [inline app/views/entities/show.rb]: This should be moved up to the top of the class.

**Pull request merged**